Guide/installation.markdown

```diff
@@ -112,6 +112,14 @@ Installing nix for IHP was done using [this guide](https://nathan.gs/2019/04/12/
 
 Note that nix can gradually grow to use several GB of disk space, especially after upgrading IHP. You can always run `nix-collect-garbage` in a `nix-shell` which will delete older/unused files.
 
+### Enabling flakes
```

Comment:
Maybe we can add, that this step is optional. If you disable flakes in your nix.conf, you can still run `ihp-new` and then still use the project (I've tried this yesterday myself)

Reply:
Yes when direnv is working, there's no manual need for `nix develop`

Reply:
I added some explaination there, what do you think?